src/driver.rs

```diff
@@ -271,7 +271,7 @@ pub fn main() {
                 },
                 _ => Some(s.to_string()),
             })
-            .chain(vec!["--cfg".into(), r#"feature="cargo-clippy""#.into()])
+            .chain(vec!["--cfg".into(), "clippy".into()])
```

Comment:
```suggestion
            .chain(vec!["--cfg".into(), r#"feature="cargo-clippy""#.into()])
            .chain(vec!["--cfg".into(), "clippy".into()])
```

For at least 1 release cycle, we should keep both. Otherwise this will break the code of Clippy users and the lint doesn't really help, except for displaying another warning in the potential sea of warnings caused by removing the `feature=cargo-clippy`.

Reply:
Ok.